docs/website/versioned_docs/version-3.0.0/command-reference/delete.md

```diff
@@ -1,40 +1,43 @@
 ---
-title: odo delete
+title: odo delete component
 sidebar_position: 4
 ---
 
-`odo delete` command is useful for deleting resources that are managed by odo.
+`odo delete component` command is useful for deleting resources that are managed by odo. It deletes the component and its related innerloop and outerloop resources from the cluster.
 
-## Deleting a component
-
-To delete a _devfile_ component, you can execute `odo delete`.
+There are 2 ways to delete a component:
+1. [Delete with access to Devfile](#delete-with-access-to-devfile)
+2. [Delete without access to Devfile](#delete-without-access-to-devfile)
 
+## Delete with access to Devfile
 ```shell
-odo delete
+odo delete component
 ```
+odo analyzes the Devfile _present in the current directory_ for the component and the outerloop resources.
+If the component has been deployed on the cluster, then odo will list all the resources and prompt the user to confirm the deletion.
```

Comment:
make sure that any references to odo you try and use `odo`. it's a pain, but we've got to try and reference `odo` like that in sentences.